style/style.css

```diff
@@ -0,0 +1,147 @@
+body {
+  margin: 0px;
+  height: 100vh;
```

Comment:
Use `min-height` instead of height, i will solve a UI problem that i still there 😄